pages/profile/[name]/[[...tab]]/index.tsx

```diff
@@ -125,19 +128,38 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
     }),
     getFollowData(user._id.toString(), reqUser),
     LevelModel.countDocuments({ isDeleted: { $ne: true }, isDraft: false, userId: userId }),
+    profileTab === ProfileTab.Levels ? LevelModel.aggregate([
```

Comment:
Should we not even both querying if reqUser is null?

Reply:
good catch